app/LuksoWebComponentsProvider.tsx

```diff
@@ -0,0 +1,17 @@
+// components/LuksoProvider.tsx
+'use client';
+
+import React, { useEffect } from 'react';
+
+export default function LuksoWebComponentsProvider({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  useEffect(() => {
+    // Dynamically import the library on the client side
+    import('@lukso/web-components');
+  }, []);
+
+  return <>{children}</>;
+}
```

Comment:
Not sure what is idea of this provider, feels like syntactic sugar. I would remove this and move import to layout

Reply:
this provider was introduced by me this there is no nice way to import as global import from doing like:
if (typeof window !== 'undefined') {
  import('@lukso/web-components');
}
we can only do like this. since in SSR it will try to import the webcomponents and that will not be possible before version 15 and app router we would just do as I mentioned above.. not on this version we need on useEffect.. because useEffect only run on client side. So to isolate this and to make clean I just created this provider.

Reply:
Perfect! let's keep it simple as possible